src/activities/PhotoHunter/Observe.qml

```diff
@@ -0,0 +1,189 @@
+
+    sourceSize.width: background.vert ? undefined : (background.width - 30) / 2
+    sourceSize.height: background.vert ?
+                           (background.height - background.barHeight - 40) /2 :
+                           background.height - background.barHeight - 30
+
+    property GCAudio audioEffects: activity.audioEffects
+    property alias repeater: repeater
+    property alias circleRepeater: circleRepeater
+    property int good: 0
+    property bool show: false
+
+    Image {
+        id: wrong
+        source: Activity.url + "wrong.svg"
+        width: 70
+        height: 70
+        opacity: 0
+    }
+
+    NumberAnimation {
+        id: wrongAnim;
+        target: wrong;
+        property: "opacity";
+        from: 0; to: 1;
+        duration: 400
+    }
+
+    NumberAnimation {
+        id: wrongAnim2;
+        target: wrong;
+        property: "opacity";
+        from: 1; to: 0;
+        duration: 400
+    }
+
+    MouseArea {
+        id: big
+        anchors.fill: parent
+        onClicked: {
+            audioEffects.play('qrc:/gcompris/src/core/resource/sounds/brick.wav')
+            wrongAnim.start()
+            wrong.x = mouseX - wrong.width/2
+            wrong.y = mouseY - wrong.height/2
+            wrongAnim2.start()
+        }
+    }
+
+    Repeater {
+        id: repeater
+
+        model: items.model
+
+        Image {
+            id: photo
+            property alias particleLoader: particleLoader
+            property alias differenceAnimation: differenceAnimation
+            property double widthScale
+            property double heightScale
+
+            width: card.width / 10 * widthScale
+            height: card.height / 10 * heightScale
+
+            opacity: 1
+
+            x: modelData[0] * card.width / 1200
+            y: modelData[1] * card.height / 1700
+
+            NumberAnimation {
+                id: differenceAnimation;
+                target: photo;
+                property: "opacity";
+                from: 0; to: 1;
+                duration: 500
+            }
+
+
+            // Create a particle only for the strawberry
+            Loader {
+                id: particleLoader
+                anchors.fill: parent
+                active: false
+                sourceComponent: particle
+            }
+
+            Component {
+                id: particle
+                ParticleSystemStarLoader {
+                    id: particles
+                    clip: false
+                }
+            }
+
+            function showParticles(item,index) {
+                item.repeater.itemAt(index).particleLoader.active = active
+                item.repeater.itemAt(index).particleLoader.item.burst(40)
+            }
+
+            MouseArea {
+                id: mouseArea
+                anchors.fill: parent
+                onClicked: {
+                    //only if the difference is not yet spotted
+                    if (img2.repeater.itemAt(index).opacity === 0) {
```

Comment:
you should not assume that your images are called img1 and img2. You should call a function in the Activity which will do it with items.img1 and items.img2